### PR TITLE
Removing Black Background for Seniors on Delegate Page

### PR DESCRIPTION
**Pull request by ando527**
changed the background colour from black to light grey to match similar styling on tables elsewhere on the site. This also required a font colour change. I also increased the readability of the text by removing the bold font weight and replacing it with a slightly larger font.

### CODE REVIEW

File: WcaOnRails/app/assets/stylesheets/delegates.scss [hunk 1]
```diff
@@ -1,7 +1,8 @@
 tr.senior-delegate {
-  background-color: black;
-  color: white;
-  font-weight: bold;
+  background-color: #e9e9e9;
+  color: black;
+  //font-weight: bold;
```
Comment (FinnIckler) [inline WcaOnRails/app/assets/stylesheets/delegates.scss:4]: why did you comment this out instead of removing it?

Comment (ando527) [reply WcaOnRails/app/assets/stylesheets/delegates.scss:4]: Sorry, bad habit 🤣, Ill fix that now

**Review by FinnIckler — APPROVED** [review]
Definitely looks better, especially when the board is the senior delegate, Thanks!

**Comment by dunkOnIT** [comment]
Way more readable, thanks a lot @ando527 ! Just one small note - if you could tag the issue in your PR description that would be awesome :D

**Pull request merged**